### PR TITLE
Move userbuffer to PyTorch

### PR DESCRIPTION
**Pull request by ksivaman**

### CODE REVIEW

**Comment by timmoon10** [comment]
Changes in https://github.com/NVIDIA/TransformerEngine/pull/161 have been merged here.

**Review by timmoon10 — APPROVED** [review]

File: transformer_engine/pytorch/csrc/userbuffers/userbuffers-host.cpp [hunk 1]
```diff
@@ -13,12 +13,11 @@
 #include <sched.h>
 #include <stdio.h>
 #include <string.h>
-#include <transformer_engine/userbuffers.h>
-#include <transformer_engine/logging.h>
```
Comment (ptrendx) [inline transformer_engine/pytorch/csrc/userbuffers/userbuffers-host.cpp:17]: Why not use logging.h?

Comment (ksivaman) [reply transformer_engine/pytorch/csrc/userbuffers/userbuffers-host.cpp:17]: This way userbuffers can be a standalone lib not dependent on libtransformerengine in any way

**Review by ptrendx — APPROVED** [review]
LGTM

**Pull request merged**